.github/ISSUE_TEMPLATE/membership-request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Membership Request
+about: New Membership for Clusterpedia
+title: "[MEMBERSHIP REQUEST] New Member of Clusterpedia"
+labels: kind/membership
+assignees: Iceber
+
+---
+
+I'm requesting to be a Member of Clusterpedia Community.
+
+## List of my contributions to the Clusterpedia project
+### PRs authored
+<!--
+As a Member, we expect you to have merged at least three pr.
+-->
+### PRs reviewed
+### Issues responded to and created
```

Comment:
`Issues responded to and created` seems relatively longer, how about `Issues involved with`

Reply:
Compared to `PRs authored` and `PRs reviewed`, it is indeed a bit longer.
@pacoxu @wzshiming Do you find `Issues involved` a bit obscure and could replace `Issues responded to or created` ?

Reply:
@windsonsea used `Issues involved`, and added some notes

Reply:
/lgtm